.clang-format

```diff
@@ -36,7 +36,7 @@ BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Linux
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: false
-ColumnLimit:     85
+ColumnLimit:     100
```

Comment:
@martin-ueding can we agree on this or should I revert? I find that long formulae often don't fit into 85 chars, while they do into 100